the-toad-tribune/src/SubApp.tsx

```diff
@@ -132,9 +135,9 @@ const SubApp = () => {
   };
 
   useEffect(() => {
-    newsDataGrabber();
-    adsDataGrabber();
-    weatherDataGrabber();
+    // newsDataGrabber();
+    // adsDataGrabber();
+    // weatherDataGrabber();
```

Comment:
is this still needed?

Reply:
Yes. Just commenting them out so we don't blow through our API keys. I will add a comment in the code letting the reviewers know for next time